index.html

```diff
@@ -301,5 +291,30 @@ <h2 class="section2-h2">skills</h2>
         </li>
       </ul>
     </section>
+
+    <!-- footer -->
+
+    <footer>
+      <h1 class="contact_title">Contact me</h1>
+      <p class="contact_paragraph">
+        If you have an application you are interested in developing, a feature
+        that you need built or a project that needs coding. I’d love to help
+        with it!
+      </p>
+      <form class="form-section">
+        <label for="name">name</label>
+        <input type="text" id="name" placeholder="enter your name" required />
+
+        <label for="email">email</label>
+        <input
+          type="email"
+          id="email"
+          placeholder="enter your email"
+          required
+        />
```

Comment:
Please add the `required` attribute to the email input so users are unable to `submit` the form until they fill in their email